docs/getting_started_introduction.md

```diff
@@ -12,11 +12,17 @@ Within the folder `users` is a directory for each user. This is a place for user
 
 ### Keyboard Project Structure
 
-Within the folder `keyboards` and its subfolder `handwired` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard`. Within it you'll find the following structure:
+Within the folder `keyboards`, its subfolder `handwired` and its vendor and manufacture subdirectories e.g. `clueboard` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard/2x1800`. Within it you'll find the following structure:
```

Comment:
Was gonna let this one go, but nope, it still bothers me (not your fault; it's been like this for a while).

```suggestion
Within the folder `keyboards`, its subfolder `handwired` and its vendor and manufacture subdirectories e.g. `clueboard` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard/2x1800`. Within it, you'll find the following structure:
```